package.json

```diff
@@ -75,11 +77,12 @@
     "@gnosis.pm/owl-token": "^3.1.0",
     "@gnosis.pm/solidity-data-structures": "^1.2.4",
     "@gnosis.pm/util-contracts": "^2.0.6",
+    "@openeth/truffle-typings": "^0.0.1",
```

Comment:
I'm curious why this necessary? The `@truffle/*` packages seem to typings - I'm assuming something is missing from them?

Reply:
the typechain generator creates types which extends this specific truffle type package. The generated code adds the specific per contract methods making it typesafe to invoke functions like `getRemainingSecondsInBatch()`